### PR TITLE
[beta] Switch to dev-static bootstrap

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
Finishes master to beta promotion.

### CODE REVIEW

File: src/stage0.txt [hunk 1]
```diff
@@ -12,15 +12,15 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
```
Comment (RalfJung) [inline src/stage0.txt:13]: This comment seems outdated? At least on the current stable branch and also in this PR I see `1.x.0` and `0.(x+1).0`, unlike what the comment says.

Comment (Mark-Simulacrum) [reply src/stage0.txt:13]: Hm yeah probably, I just go off of the forge docs. Mind filing a PR against master fixing this? (Or just an issue is fine too.)

**Comment by Mark-Simulacrum** [comment]
@bors r+

I've pushed up a commit that should let miri not build on the dist builders. I'm unclear why this wasn't a problem before now, but possibly we just happened to always have miri building...

**Comment by Dylan-DPC** [comment]
@bors retry

bors didn't fire last time

**Comment by Mark-Simulacrum** [comment]
cc @topecongiro I've pushed up a rust-1.44.0-beta branch to the rustfmt repository and cherry-picked the test fixes onto it. It would be great to get that into rust-lang/rust's master rustfmt; we've had to apply this patch for a number of beta releases at this point. Let me know if I can help do that.

@bors r+

**Pull request merged**